orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModel.java

```diff
@@ -0,0 +1,135 @@
+      new OrchestrationAiModel("meta--llama3-70b-instruct");
+
+  /** Meta Llama3.1 70B Instruct model */
+  public static final OrchestrationAiModel LLAMA3_1_70B_INSTRUCT =
+      new OrchestrationAiModel("meta--llama3.1-70b-instruct");
+
+  /** Anthropic Claude 3 Sonnet model */
+  public static final OrchestrationAiModel CLAUDE_3_SONNET =
+      new OrchestrationAiModel("anthropic--claude-3-sonnet");
+
+  /** Anthropic Claude 3 Haiku model */
+  public static final OrchestrationAiModel CLAUDE_3_HAIKU =
+      new OrchestrationAiModel("anthropic--claude-3-haiku");
+
+  /** Anthropic Claude 3 Opus model */
+  public static final OrchestrationAiModel CLAUDE_3_OPUS =
+      new OrchestrationAiModel("anthropic--claude-3-opus");
+
+  /** Anthropic Claude 3.5 Sonnet model */
+  public static final OrchestrationAiModel CLAUDE_3_5_SONNET =
+      new OrchestrationAiModel("anthropic--claude-3.5-sonnet");
+
+  /** Amazon Titan Text Lite model */
+  public static final OrchestrationAiModel TITAN_TEXT_LITE =
+      new OrchestrationAiModel("amazon--titan-text-lite");
+
+  /** Amazon Titan Text Express model */
+  public static final OrchestrationAiModel TITAN_TEXT_EXPRESS =
+      new OrchestrationAiModel("amazon--titan-text-express");
+
+  /** Azure OpenAI GPT-3.5 Turbo chat completions model */
+  public static final OrchestrationAiModel GPT_35_TURBO = new OrchestrationAiModel("gpt-35-turbo");
+
+  /** Azure OpenAI GPT-3.5 Turbo chat completions model */
+  public static final OrchestrationAiModel GPT_35_TURBO_16K =
+      new OrchestrationAiModel("gpt-35-turbo-16k");
+
+  /** Azure OpenAI GPT-4 chat completions model */
+  public static final OrchestrationAiModel GPT_4 = new OrchestrationAiModel("gpt-4");
+
+  /** Azure OpenAI GPT-4-32k chat completions model */
+  public static final OrchestrationAiModel GPT_4_32K = new OrchestrationAiModel("gpt-4-32k");
+
+  /** Azure OpenAI GPT-4o chat completions model */
+  public static final OrchestrationAiModel GPT_4O = new OrchestrationAiModel("gpt-4o");
+
+  /** Azure OpenAI GPT-4o-mini chat completions model */
+  public static final OrchestrationAiModel GPT_4O_MINI = new OrchestrationAiModel("gpt-4o-mini");
+
+  /** Google Cloud Platform Gemini 1.0 Pro model */
+  public static final OrchestrationAiModel GEMINI_1_0_PRO =
+      new OrchestrationAiModel("gemini-1.0-pro");
+
+  /** Google Cloud Platform Gemini 1.5 Pro model */
+  public static final OrchestrationAiModel GEMINI_1_5_PRO =
+      new OrchestrationAiModel("gemini-1.5-pro");
+
+  /** Google Cloud Platform Gemini 1.5 Flash model */
+  public static final OrchestrationAiModel GEMINI_1_5_FLASH =
+      new OrchestrationAiModel("gemini-1.5-flash");
+
+  OrchestrationAiModel(@Nonnull final String modelName) {
+    this.modelName = modelName;
+  }
+
+  @Nonnull
+  LLMModuleConfig createConfig() {
+    return new LLMModuleConfig()
+        .modelName(modelName)
+        .modelParams(modelParams)
```

Comment:
currently, I'd expect this to cause failures, because `modelParams` is required, but is null here by default, right?